README.md

```diff
@@ -6,10 +6,6 @@ This app is a <em>minimalistic</em> C++ example that shows how to use [WebGPU](h
 
 On the web, the app is built against [emdawnwebgpu](https://dawn.googlesource.com/dawn/+/refs/heads/main/src/emdawnwebgpu/) (Emscripten Dawn WebGPU), which has bindings implementing webgpu.h on top of the JavaScript API. It uses [Emscripten](https://emscripten.org/), a tool for compiling C/C++ programs to WebAssembly. On specific platforms such as macOS or Windows, this project can be built against [Dawn](https://dawn.googlesource.com/dawn/), Chromium's cross-platform WebGPU implementation.
 
-<p align="center" width="100%">
-:warning: The webgpu.h and webgpu_cpp.h APIs are not yet stabilized. :warning:
```

Comment:
Should perhaps still warn that only the core API is stable. Doesn't need to be as "loud" as this though :)

Reply:
I've amended this PR. Please let me know what you think

Reply:
I don't think it should be part of the standard `webgpu.h` because its behavior depends on some internals of Emscripten. They're not super complicated but very much an Emscripten-specific thing, with extra features like [specialHTMLTargets](https://emscripten.org/docs/api_reference/html5.h.html#registration-functions) to custom register fake "selector" names.

Reply:
But like I said we could declare it a stable part of Emdawnwebgpu.